.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
```

Comment:
`@netlify/build` uses `snake_case` instead of `kebab-case`. We might want to harmonize here.
`.*.md` is to allow `README.md` and the like to be uppercase. We could also lowercase them.